sphinx/util/pycompat.py

```diff
@@ -103,3 +103,40 @@ def execfile_(filepath, _globals):
         else:
             raise
     exec_(code, _globals)
+
+# ------------------------------------------------------------------------------
+# Internal module backwards-compatibility
+
+import warnings
+
+from six import class_types
```

Comment:
@lehmannro How do you think for my comment? If you are ok, I'm going to merge this for 1.3b3.